packages/transformer/src/IModelImporter.ts

```diff
@@ -155,6 +175,16 @@ export class IModelImporter {
     return false;
   }
 
+  /**
+   * Marks an element so that it can be updated during import when [[IModelImportOptions.preserveElementIdsForFiltering]] is set to true
+   */
+  public markElementAsUpdate(elementId: Id64String) {
+    if (this._rootElementIds.has(elementId)) {
+      return;
+    }
+    this._elementsToUpdate.add(elementId);
+  }
+
```

Comment:
- Thanks for the suggestions
- Removed `fromTransformation` and `preserveElementIdsForFiltering` importer options.
- Updated the names to `elementIdsToUpdateForPreserveId` and `markElementToUpdateForPreserveId` to better reflect their functions (let me know if these names work):
  - `elementIdsToUpdateForPreserveId` tracks the element IDs to update when `preserveElementIdsForFiltering` is set to true.
  - `markElementToUpdateForPreserveId` is called when `targetElementId` is valid (indicating an update) during export in transformation, adding the element to the `elementIdsToUpdateForPreserveId` set.
- We don't need to track elements to insert, as we can determine whether to preserve an element ID during insert with the `forceUseId` flag. If the set is defined, it indicates that the ID should be preserved. The set is initialized when the `preserveElementIdsForFiltering` option is set to true, if it is undefined.

![image](https://github.com/user-attachments/assets/5e708a07-bc5f-4b28-8498-dd77fd1fa805)

Reply:
- I think it is great that `fromTransformation` is gone
- I see that `preserveElementIdsForFiltering` was re-implemented as `enablePreserveElementIds()`. My initial assumption was that we can get rid of `preserveElementIdsForFiltering ` but since it is not true, can we revert this part to have `preserveElementIdsForFiltering` instead of `enablePreserveElementIds` as it was before. I think there is not enough value in this change, to justify this breaking change to user.